packages/bpk-theming/readme.md

```diff
@@ -0,0 +1,45 @@
+# bpk-theming
+
+> Backpack theming utilities.
+
+## Installation
+
+```sh
+npm install bpk-theming --save-dev
+```
+
+## Usage
+
+```js
+import BpkThemeProvider from 'bpk-theming';
+
+import BpkButton, {themeAttributes as buttonThemeAttributes} from 'bpk-component-button';
+
+const theme = {
+  buttonPrimaryGradientStartColor: '#fce134',
+  buttonPrimaryGradientEndColor: '#f8c42d',
+  buttonPrimaryTextColor: '#2d244c',
+  buttonSecondaryBackgroundColor: '#fff',
+  buttonSecondaryTextColor: '#2d244c',
+  buttonSecondaryBorderColor: '#2d244c',
+};
+
+export default class App extends Component {
+  render() {
+    return (
+      <BpkThemeProvider theme={theme} themeAttributes={buttonThemeAttributes}>
```

Comment:
`buttonThemeAttributes` should be in an array

Reply:
It's an array itself though, so `[...buttonThemeAttributes]` would be redundant, unless it's just for clarity to users.

Reply:
No, but when you have multiple we'd want to have `themeAttributes={[ buttonThemeAttributes, linkThemeAttributes ]}` so we should be consistent and always have the top level array

Reply:
Fair enough, will change